assets/src/utils/common.ts

```diff
@@ -31,6 +33,14 @@ function hasContent(item: any): boolean {
     if (!item) return false;
     if (typeof item?.type === 'string' && item.type !== 'p') return true;
     if (Array.isArray(item)) return item.some(hasContent);
+
+    if (item?.type) {
+      const s = schema[item.type as ModelTypes];
+      const sc = schema;
+      // We'll assume void elements are content.
+      if (s?.isVoid) return true;
+    }
```

Comment:
On the client, I could rely on the element schemas, but over in serverland, I ended up with a static list of elements since there's no concept of void-elements there.